test/test_foreach.py

```diff
@@ -355,6 +357,7 @@ def clone(arg):
 
     @ops(foreach_pointwise_op_db)
     @parametrize("is_fastpath", (True, False))
+    @unittest.skipIf(_get_torch_cuda_version() >= (12, 6), "This test is failing on CUDA 12.6")
```

Comment:
Cna you file an issue to track this please? With the details of the error so that we can debug